joomla/joomla-cms
# [4.0] Replace JS Diff script

Pull Request for Issue #14592

Also means https://github.com/joomla/joomla-cms/pull/14611 should be merged.

### Summary of Changes

This PR introduces a vanilla JS diff script for the content version component, with some HTML5 fixes and other improvements to make the view more simplistic.

### Testing Instructions

- Apply PR
- Ensure you have content versioning enabled
- Update any article
- Click "Versions" in the toolbar
- Select 2 articles and click "Compare"

![screeny](https://cloud.githubusercontent.com/assets/2019801/23908282/781636ec-08cb-11e7-957f-8a9ecf0c7fb8.png)
